tools/benchmark_tool/openvino/tools/benchmark/parameters.py

```diff
@@ -77,7 +77,13 @@ def parse_args():
                            'Show progress bar (can affect performance measurement). Default values is \'False\'.')
     args.add_argument('-shape', type=str, required=False, default='',
                       help='Optional. '
-                           'Set shape for input. For example, "input1[1,3,224,224],input2[1,4]" or "[1,3,224,224]" in case of one input size.')
+                           'Set shape for input. For example, "input1[1,3,224,224],input2[1,4]" or "[1,3,224,224]" in case of one input size.'
+                           'This parameter affect model Parameter shape, can be dynamic. For dynamic dimesions use symbol `?`, `-1` or range `low.. up`.')
+    args.add_argument('-tensor_shape', type=str, required=False, default='',
```

Comment:
To make it more clear I suggest rephrasing this to:
- for "-shape": "Set shape for model inputs..."
- change "-tensor_shape" to "-data_shape": "Set shape for input data..."

Reply:
@nkogteva what do you think?

Reply:
@AlexeyLebedev1 It makes sense. The description should be updated in C++ to be aligned cc @fzhar @ivikhrev